### PR TITLE
Add function getYears, getMonths, getDays

### PR DESCRIPTION
**Pull request by thaianhtaivn**
Already tested function with platformIO & ESP8266.

### CODE REVIEW

**Comment by per1234** [comment]
Alternative to:

- https://github.com/arduino-libraries/NTPClient/pull/60
- https://github.com/arduino-libraries/NTPClient/pull/161

**Comment by thaianhtaivn** [comment]
Hello @per1234 ,
Because we usually use it, can we merge these functions to master?
Thank you!

**Comment by per1234** [comment]
Hi @thaianhtaivn. Thanks for your pull request. My role in this project is somewhat janitorial in nature. I don't make the decisions about which pull requests are merged or not. I commented to provide some supplemental information that might be of use to the reviewers.

Regards,
Per

File: NTPClient.cpp [hunk 1]
```diff
@@ -148,6 +148,27 @@ int NTPClient::getMinutes() const {
 int NTPClient::getSeconds() const {
   return (this->getEpochTime() % 60);
 }
+int NTPClient::getYears() const {
+    time_t rawtime = this->getEpochTime();
```
Comment (lukaszgieraltowski) [inline NTPClient.cpp:152]: Use two spaces for indentation instead of four.

File: NTPClient.h [hunk 2]
```diff
@@ -85,6 +85,9 @@ class NTPClient {
     int getHours() const;
     int getMinutes() const;
     int getSeconds() const;
+	int getYears() const;
```
Comment (lukaszgieraltowski) [inline NTPClient.h:88]: Use two spaces for indentation instead of tab.

**Review by lukaszgieraltowski — CHANGES_REQUESTED** [review]
Good job. I'd suggest to add `getFormattedDateTime` using new methods. Analogously to `getFormattedTime`.

**Comment by lukaszgieraltowski** [comment]
@thaianhtaivn Actually this was already implemented https://github.com/arduino-libraries/NTPClient/pull/94 but not merged :/

**Pull request closed without merging**